src/python/py_vlm_pipeline.cpp

```diff
@@ -235,6 +239,23 @@ void init_vlm_pipeline(py::module_& m) {
             py::arg("streamer") = std::monostate(), "streamer",
             (vlm_generate_docstring + std::string(" \n ")).c_str()
         )
+        .def(
+            "generate",
+            [](ov::genai::VLMPipeline& pipe,
+                const std::string& prompt,
+                const std::vector<ov::Tensor>& video,
+                const ov::genai::GenerationConfig& generation_config,
+                const pyutils::PyBindStreamerVariant& streamer,
+                const py::kwargs& kwargs
+            ) -> py::typing::Union<ov::genai::VLMDecodedResults> {
+                return call_vlm_generate(pipe, prompt, video, generation_config, streamer, kwargs, true);
+            },
+            py::arg("prompt"), "Input string",
+            py::arg("video"), "Input video",
```

Comment:
I believe it should be possible to mix images and videos as stated in ticket 171515. By the way, add a test for that comparing with optimum-intel